examples/net_framework/CSharpExamples/DeepSpeechConsole/Program.cs

```diff
@@ -38,13 +67,18 @@ static void Main(string[] args)
                 {
                     Console.WriteLine($"Model loaded - {stopwatch.Elapsed.Milliseconds} ms");
                     stopwatch.Reset();
-                    Console.Write("Enable LM [y/n]:");
-                    if (Console.ReadLine().ToLower() == "y")
+                    if (lm == null)
+                        Console.Write("Enable LM [y/n]:");
```

Comment:
That feels weird, but it's your call ; though it'd be better we get consistency with all other CLI tools, so no interactive behavior if possible :)

Reply:
Removed